### ISSUE TITLE
Alpha: UsageRecordResource.Read with start and end parameters set fails

### ISSUE THREAD

**Issue by jonastelzio** [twilio/twilio-csharp#372]
**Version: 5.5.1-alpha1** 
**API Subdomain (api/taskrouter/ip_messaging): wireless**

### Code Snippet
```c#
TwilioClient.Init(".....", ".....");
var from = DateTime.UtcNow.AddDays(-3);
var to = from.AddMonths(1);
var usage = UsageRecordResource.Read("<redacted>", start: from, end: to);
```

### Exception / Log
```
Invalid Start: 2017-06-12T09:48:02. Start must be a valid ISO8601 formatteddatetime string: yyyy-MM-dd'T'HH:mm:ss'Z'
```

### Steps to Reproduce
1. Use Alpha Version
2. Initialize a Twilio Client
3. Attempt to read a UsageRecordResource with start and end parameters defined

This is probably just an invalid date.ToString(....) format string. I think using .ToString("o") will do the trick.

**Comment by jmctwilio** [comment]
@jonastelzio Looks like the standard for ISO8601 for C# is:

`DateTime.ToString("s", System.Globalization.CultureInfo.InvariantCulture);`

**Comment by jmctwilio** [comment]
#374

**Comment by efossier** [comment]
@jonastelzio This should be fixed in `5.5.2` and `5.5.3-alpha1`
